src/internal/components/drag-handle-wrapper/__tests__/portal-overlay.test.tsx

```diff
@@ -59,3 +59,45 @@ test('matches the position of the tracked element in rtl', async () => {
     expect(portalOverlay.style.height).toBe('20px');
   });
 });
+
+test('does not update position when disabled', async () => {
+  const trackElement = document.createElement('span');
+
+  render(
+    <PortalOverlay track={trackElement} isDisabled={true}>
+      <div id="overlay">Overlay</div>
+    </PortalOverlay>
+  );
+
+  const portalOverlay = document.querySelector<HTMLElement>(`.${styles['portal-overlay']}`)!;
+  await waitFor(() => {
+    expect(portalOverlay.style.translate).toBeUndefined();
+    expect(portalOverlay.style.width).toBe('');
+    expect(portalOverlay.style.height).toBe('');
+  });
+});
+
+test('resumes position updates when enabled after being disabled', async () => {
+  const trackElement = document.createElement('span');
+
+  const PortalOverlayWrapper = ({ isDisabled }: { isDisabled: boolean }) => (
+    <PortalOverlay track={trackElement} isDisabled={isDisabled}>
+      <div id="overlay">Overlay</div>
+    </PortalOverlay>
+  );
+
+  const { rerender } = render(<PortalOverlayWrapper isDisabled={true} />);
+  const portalOverlay = document.querySelector<HTMLElement>(`.${styles['portal-overlay']}`)!;
+  await waitFor(() => {
+    expect(portalOverlay.style.translate).toBeUndefined();
+    expect(portalOverlay.style.width).toBe('');
+    expect(portalOverlay.style.height).toBe('');
+  });
+
+  rerender(<PortalOverlayWrapper isDisabled={false} />);
+  await waitFor(() => {
+    expect(portalOverlay.style.translate).toBe('-2px 4px');
```

Comment:
Why is it `-2px` instead of `2px` as in line 39?

Reply:
Because `isRtl` is set to `true` in line 46 and did not get reset anymore.

That's not nice, let me fix that.